packages/typespec-ts/test/integration/azurecore.spec.ts

```diff
@@ -92,42 +92,37 @@ describe("Azure Core Rest Client", () => {
   });
 
   it("should list users", async () => {
-    try {
-      const result = await client.path("/azure/core/basic/users").get({
-        queryParameters: {
-          top: 5,
-          skip: 10,
-          orderby: "id",
-          filter: "id lt 10",
-          select: buildMultiCollection(["id", "orders", "etag"], "select"),
-          expand: "orders"
-        },
-        skipUrlEncoding: true
-      });
-      if (isUnexpected(result)) {
-        throw Error("Unexpected status code");
+    const result = await client.path("/azure/core/basic/users").get({
+      queryParameters: {
+        top: 5,
+        skip: 10,
+        orderby: buildExplodedFormStyle(["id"]),
+        filter: "id lt 10",
+        select: buildExplodedFormStyle(["id", "orders", "etag"]),
+        expand: buildExplodedFormStyle(["orders"])
       }
-      const responseBody = {
-        value: [
-          {
-            id: 1,
-            name: "Madge",
-            etag: "11bdc430-65e8-45ad-81d9-8ffa60d55b59",
-            orders: [{ id: 1, userId: 1, detail: "a recorder" }]
-          },
-          {
-            id: 2,
-            name: "John",
-            etag: "11bdc430-65e8-45ad-81d9-8ffa60d55b5a",
-            orders: [{ id: 2, userId: 2, detail: "a TV" }]
-          }
-        ]
-      };
-      assert.strictEqual(result.status, "200");
-      assert.deepEqual(result.body, responseBody);
-    } catch (err) {
-      assert.fail(err as string);
+    });
+    if (isUnexpected(result)) {
+      throw Error("Unexpected status code");
     }
+    const responseBody = {
+      value: [
+        {
+          id: 1,
+          name: "Madge",
+          etag: "11bdc430-65e8-45ad-81d9-8ffa60d55b59",
+          orders: [{ id: 1, userId: 1, detail: "a recorder" }]
+        },
+        {
+          id: 2,
+          name: "John",
+          etag: "11bdc430-65e8-45ad-81d9-8ffa60d55b5a",
+          orders: [{ id: 2, userId: 2, detail: "a TV" }]
+        }
+      ]
+    };
+    assert.strictEqual(result.status, "200");
+    assert.deepEqual(result.body, responseBody);
   });
 
```

Comment:
why remove the catch error part in this case? what about other cases?

Reply:
removed useless codes here. we don't need to catch any error to fail. if exception happens the case would fail directly.

i updated cases i met in this pr. but i remember i told jiaodi to remove for any new cases.